src/ConnectedMode/ConnectedModePackage.cs

```diff
@@ -63,6 +64,7 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             serverSuppressionsChangedHandler = componentModel.GetService<ServerSuppressionsChangedHandler>();
             boundSolutionUpdateHandler = componentModel.GetService<BoundSolutionUpdateHandler>();
             timedUpdateHandler = componentModel.GetService<TimedUpdateHandler>();
+            localSuppressionsChangedHandler = componentModel.GetService<LocalSuppressionsChangedHandler>();
```

Comment:
The new class is MEF-created by the `ConnectedMode` package. It isn't referenced anywhere else, and we don't test the `ConnectedMode` package initialization, so I didn't create an interface for the type (same as for the other three new types in the lines immediately above).